media/js/ie8/mozilla-utils.js

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Create namespace
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+(function() {
+    'use strict';
+
+    var Utils = {};
+
+    /**
+     * Bug 393263 A special function for IE < 9.
+     * Without this hack there is no prompt to download after they click. sigh.
+     * @param {link} direct link to download URL
+     * @param {userAgent} optional UA string for testing purposes.
+     */
+    Utils.triggerIEDownload = function(link, userAgent) {
```

Comment:
Should we remove these functions from the regular `mozilla-utils.js` and then run the unit tests against this file?

Reply:
Good call. Will do.